tasks/apply-mapping/tests/test-apply-mapping-other-formats.yaml

```diff
@@ -0,0 +1,96 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-apply-mapping-other-formats
+spec:
+  description: |
+    Run the apply-mapping task with a snapshot.spec json where the repository is in a format not specifically
+    handled by the task. Verify that the resulting json remains unchanged in the repository field.
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup-other-formats
+      workspaces:
+        - name: config
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: config
+        steps:
+          - name: setup-other-formats-values
+            image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+            script: |
+              #!/usr/bin/env sh
+              set -eux
+
+              cat > $(workspaces.config.path)/test_data_other_formats.json << EOF
+              {
+                "mapping": {
+                  "components": [
+                    {
+                      "name": "comp1",
+                      "repository": "quay.io/my-sample/my-image"
+                    }
+                  ]
+                }
+              }
+              EOF
+
+              cat > $(workspaces.config.path)/test_snapshot_spec_other_formats.json << EOF
+              {
+                "application": "myapp",
+                "components": [
+                  {
+                    "name": "comp1",
+                    "containerImage": "imageurl1@sha256:123456",
+                    "repository": "quay.io/my-sample/my-image"
```

Comment:
A snapshot isn't going to have a repository value in it pre apply-mapping, only the data file will

Reply:
ah good catch, thanks.